apps/cowswap-frontend/src/modules/hooksStore/hookRegistry.tsx

```diff
@@ -2,12 +2,13 @@ import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
 import { PRE_BUILD, POST_BUILD } from './dapps/BuildHookApp/hook'
 import { PRE_CLAIM_GNO } from './dapps/ClaimGnoHookApp/hook'
+import { PRE_PERMIT } from './dapps/PermitHookApp/hook'
 import { HookDapp } from './types/hooks'
 
 export const PRE_HOOK_REGISTRY: Record<SupportedChainId, HookDapp[]> = {
   [SupportedChainId.MAINNET]: [PRE_BUILD],
   [SupportedChainId.GNOSIS_CHAIN]: [PRE_CLAIM_GNO, PRE_BUILD],
-  [SupportedChainId.SEPOLIA]: [PRE_BUILD],
+  [SupportedChainId.SEPOLIA]: [PRE_BUILD, PRE_PERMIT],
```

Comment:
Since this is not necessarily related to the SWAP, this can be a post hook as well 🤷